PurchasesTests/Networking/BackendTests.swift

```diff
@@ -1959,20 +1880,23 @@ private extension BackendTests {
             "other_purchases": [:]
         ]
     ]}
+
 }
 
 private extension BackendTests.RequestCall {
 
-    func expectToEqual(_ other: BackendTests.RequestCall, file: FileString = #file, line: UInt = #line) {
-        switch other.request.method {
-        case .get:
-            expect(file: file, line: line, self.request.requestBody).to(beNil())
+    // fixme: use SnapshotTesting to compare the whole `HTTPRequest` instead of only `requestBody`.
+    func expectToEqual(_ other: BackendTests.RequestCall, file: FileString = #file, line: UInt = #line) throws {
```

Comment:
same question, do we already have a PR or a jira for this? Just wanna make sure we don't forget about it

Reply:
This is done in #1366